src/main/kotlin/com/hedvig/accountservice/integration/paymentService/PaymentServiceClient.kt

```diff
@@ -18,4 +20,9 @@ interface PaymentServiceClient {
         @PathVariable("memberId") memberId: String,
         @RequestBody chargeRequestDto: ChargeRequestDto
     ): ResponseEntity<UUID>
+
+    @GetMapping("/v2/_/members/connectedToProvider/{paymentProvider}")
+    fun getMembersConnectedToProvider(
+        @PathVariable("paymentProvider") paymentProvider: PaymentProvider
```

Comment:
Super subjective taste question
```suggestion
        @PathVariable("provider") provider: PaymentProvider
```